run_kraken.py

```diff
@@ -382,14 +381,16 @@ def main(cfg):
             telemetry_k8s.collect_cluster_metadata(chaos_telemetry)
 
         decoded_chaos_run_telemetry = ChaosRunTelemetry(json.loads(chaos_telemetry.to_json()))
-        logging.info(f"Telemetry data:\n{decoded_chaos_run_telemetry.to_json()}")
+        chaos_output.telemetry = decoded_chaos_run_telemetry
+        logging.info(f"Chaos data:\n{chaos_output.to_json()}")
         telemetry_elastic.upload_data_to_elasticsearch(decoded_chaos_run_telemetry.to_json(), elastic_index)
         if config["telemetry"]["enabled"]:
             logging.info(f"telemetry data will be stored on s3 bucket folder: {telemetry_api_url}/download/{telemetry_request_id}")
             logging.info(f"telemetry upload log: {safe_logger.log_file_name}")
             try:
                 telemetry_k8s.send_telemetry(config["telemetry"], telemetry_request_id, chaos_telemetry)
                 telemetry_k8s.put_cluster_events(telemetry_request_id, config["telemetry"], start_time, end_time)
+                telemetry_k8s.put_critical_alerts(telemetry_request_id, config["telemetry"], summary)
```

Comment:
I'm not sure if the alert is in the right place in the telemetry. Also should **scenario** and **run_id** be outside of the _critical_alerts_ section?

```
{
....
....
"critical_alerts": {
        "chaos_alerts": [
            {
                "alertname": "NorthboundStale",
                "alertstate": "pending",
                "namespace": "openshift-ovn-kubernetes",
                "severity": "critical"
            }
        ],
        "post_chaos_alerts": [],
        "scenario": "plugin_scenarios",
        "run_id": "6ca05da4-81ac-49fd-8730-0c5005f4515c"
    }
}
```

Reply:
This thing is related to the fact that this output is printed together with the telemetry in the krkn output, but in the telemetry webservice the two files are distinct so I put the run id in the chaos alert to easily reconcile them on a post processing job, so if it doesn't bother too much, I'd keep the run id within the structure.

Reply:
gotcha, so if you run multiple scenarios you would be able to see it. Makes sense, thanks